examples/winit.rs

```diff
@@ -71,7 +71,7 @@ fn main() -> wry::Result<()> {
           event: WindowEvent::Resized(size),
           ..
         } => {
-          _webview.set_bounds(wry::Rect {
+          let _ = _webview.set_bounds(wry::Rect {
```

Comment:
LGTM overall, but I think all example should unwrap the call instead. We could find which part gets the error in this way.

Reply:
good idea, done